packages/editor/src/components/browser/CmsBrowser.tsx

```diff
@@ -1,15 +1,151 @@
+    type: 'STRING',
+    values: {}
+  });
+  const [showHelper, setShowHelper] = useState<boolean>(false);
+
+  const columns = useMemo<ColumnDef<ContentObject>[]>(
+    () => [
+      {
+        accessorFn: row => `${row.name} (${row.type})`,
+        id: 'name',
+        cell: cell => {
+          if (cell.row.original.type === 'FOLDER') {
+            return <ExpandableCell cell={cell} title={cell.row.original.name} additionalIcon={'GoToSource'} />;
+          } else {
+            return <ExpandableCell cell={cell} title={cell.row.original.name} additionalIcon={''} />;
+          }
+        }
+      }
+    ],
+    []
+  );
+
+  const [expanded, setExpanded] = useState<ExpandedState>(false);
+  const [globalFilter, setGlobalFilter] = useState('');
+  const [rowSelection, setRowSelection] = useState<RowSelectionState>({});
+
+  const table = useReactTable({
+    data: tree,
+    columns: columns,
+    state: {
+      expanded,
+      globalFilter,
+      rowSelection
+    },
+    filterFromLeafRows: true,
+    enableRowSelection: true,
+    enableMultiRowSelection: false,
+    enableSubRowSelection: false,
+    onExpandedChange: setExpanded,
+    onGlobalFilterChange: setGlobalFilter,
+    onRowSelectionChange: setRowSelection,
+    getSubRows: row => row.children,
+    getCoreRowModel: getCoreRowModel(),
+    getExpandedRowModel: getExpandedRowModel(),
+    getFilteredRowModel: getFilteredRowModel()
+  });
+
+  useEffect(() => {
+    if (Object.keys(rowSelection).length !== 1) {
+      setSelectedContentObject({ name: '', children: [], fullPath: '', type: 'STRING', values: {} });
+      setShowHelper(false);
+      onChange('');
+      return;
+    }
+    const selectedRow = table.getRowModel().rowsById[Object.keys(rowSelection)[0]];
+    setSelectedContentObject(selectedRow.original);
+    setShowHelper(true);
+    onChange(selectedRow.original.fullPath);
+  }, [onChange, rowSelection, table]);
+
+  return (
+    <>
+      <Table search={{ value: globalFilter, onChange: setGlobalFilter }}>
+        <tbody>
+          {table.getRowModel().rows.map(row =>
+            row.original.type === 'FOLDER' ? (
+              <MessageRow key={row.id}>
+                {row.getVisibleCells().map(cell => (
+                  <TableCell key={cell.id}>{flexRender(cell.column.columnDef.cell, cell.getContext())}</TableCell>
+                ))}
+              </MessageRow>
+            ) : (
+              <SelectRow key={row.id} row={row}>
+                {row.getVisibleCells().map(cell => (
+                  <TableCell key={cell.id}>{flexRender(cell.column.columnDef.cell, cell.getContext())}</TableCell>
+                ))}
+              </SelectRow>
+            )
+          )}
+        </tbody>
+      </Table>
+      <Collapsible label='Helper Text' defaultOpen={showHelper}>
+        {value.length !== 0 && selectedContentObject ? (
+          <pre className='browser-helptext'>
+            <p className='helperTitle'>{createFullPathCMS(value)}</p>
```

Comment:
There is no benefit from showing the `ivy.cms.co(...` stuff here. This is only some technical stuff and should only be added on the accept method! 
Simply show the full path here with the type, e.g. `/test/Bla (File)`